subjects/hello-world/README.md

```diff
@@ -0,0 +1,43 @@
+# hello-world
+
+| Complexity | Executable | Goal |
+|------|------|------|
+| Low | `greet` | Create a CLI Program that outputs a greeting.
+
+This subject is purely aimed to trying out the validation workflow.
+
+## Feature
+
+### Greet
+
+The program `greet` *may* have an argument to specify who to greet.
+
+```console
+$ greet Bob
+Hello Bob.
+```
+
+If no person to greet is specified, `greet` the World.
+
+```console
+$ greet
+Hello World.
+```
+
+## Additional targets
+
+??? success "Packaging"
+    Making a packaging mechanism available and your program easily installable
+    is considered a "soft requirement" as it will be required for other exercises.
+
+??? warning "Lint"
+    In order to ease the reviewing phase, please make sure your repository
+    follows the common linting guidelines for your language.
+
+??? warning "Documentation"
+    Make sure your repository is properly documented and that your code can
+    be easily understood.
+
+## Validations
+
+- None
```

Comment:
It would be interresting to add a little script for testing, to compare outputs.

Reply:
I think I will officially add "Validation scripts" on the roadmap. I still have some specifications to tackle before actually implementing this notion in open-subjects.

I'll ping you on the corresponding issue(s) as soon as I'll start working on it :rocket:

Reply:
Sure ;)